digital-literacy/README.md

```diff
@@ -1,35 +1,42 @@
+<a href="https://emergentworks.org"><img src="../EWLogo.png" alt="EmergentWorks Logo" title="EmergentWorks Logo" width="194" height="35"/></a>
+
```

Comment:
I wish there was a way to always have this in the header. Maybe when we have time, we can customize the Github Page template.

Reply:
agreed... pretty sure there's a way to do this, just haven't dived into the templating stuff